### PR TITLE
Bugfix/account management fix navigation and userdetails update

### PR DESCRIPTION
**Pull request by omarCreativeDev**
- fix navigation and userdetails update
- set back links

### CODE REVIEW

**Comment by omarCreativeDev** [comment]
[Accessibility version: where different accordion views are opened, errors from previous one should be cleared.](https://trello.com/c/Z2ZrXygo/570-accessibility-version-where-different-accordion-views-are-opened-errors-from-previous-one-should-be-cleared)

**Comment by omarCreativeDev** [comment]
[Account Management - Change Account Details link not working](https://trello.com/c/1JCpHl2f/654-account-management-change-account-details-link-not-working)

File: src/app/features/account-management/your-account/your-account.component.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h2 class="govuk-heading-m govuk-!-margin-bottom-7">Account details</h2>
           {{ user?.fullname }}
         </dd>
         <dd class="govuk-summary-list__actions">
-          <a [routerLink]="['/change-your-details']" class="govuk-link">
+          <a [routerLink]="['/registration/change-your-details']" class="govuk-link">
```
Comment (ChazUK) [inline src/app/features/account-management/your-account/your-account.component.html]: I don't think this should be directing back to the registration page, it should still be under the same route as account-management.

File: src/app/features/registration/registration-routing.module.ts [hunk 2]
```diff
@@ -50,7 +51,7 @@ const routes: Routes = [
   {
     path: 'change-your-details',
     component: ChangeYourDetailsComponent,
-    canActivate: [RegisterGuard],
+    canActivate: [AuthGuard],
```
Comment (ChazUK) [inline src/app/features/registration/registration-routing.module.ts]: Same here, Wouldn't this mean the registration flow is now broken as users aren't able to change their details?

File: src/app/features/account-management/your-account/your-account.component.html (hunk 1 above)
Comment (omarCreativeDev) [reply src/app/features/account-management/your-account/your-account.component.html]: done

File: src/app/features/registration/registration-routing.module.ts (hunk 2 above)
Comment (omarCreativeDev) [reply src/app/features/registration/registration-routing.module.ts]: this has been taken care of in my latest commit

**Pull request merged**